applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -584,7 +602,8 @@ fetch_local_resources(AccountId) ->
         {'ok', JObjs} ->
             CacheProps = [{'origin', fetch_local_cache_origin(JObjs, AccountDb, [])}],
             Resources = resources_from_jobjs([wh_json:get_value(<<"doc">>, JObj) || JObj <- JObjs]),
-            LocalResources = [Resource#resrc{global='false'} || Resource <- Resources],
+            Proxies = fetch_account_dedicated_proxies(AccountId),
```

Comment:
we are already iterating the returned resources to set global='false' and i opted to set proxies on that iteration. resources_from_jobjs is also used by fetch_global_resources that's why we have the iterator after the return, why not use that iterator to set the proxies ?